doc/howtos/software-update-server.rst

```diff
@@ -158,31 +156,31 @@ customized image, by:
    syntax. The other identifiers are under the developer's control.
 
 
-This will add, to the set of images that are buildable, one referred to as
-``ostro-image-swupd-pre_installed_content``, which contains, besides the
-os-core bundle, also the ``sudo_bundle`` bundle from Step 1 and whatever
+This will add a new image option called
+``ostro-image-swupd-pre_installed_content``
+to the set of images that are buildable. This new image
+contains the
+``os-core bundle``,  the ``sudo_bundle`` bundle from Step 1, and whatever
 else might have been added.
 
-The main difference, compared to the approach taken in Step 0 is that
-this approach allows for removing the content of the bundle without having to
+Compared to the approach taken in Step 0, 
+this approach allows you to remove the content of the bundle without having to
 create an update (and thus a new release).
```

Comment:
Reading this, both new and old version, looks confusing. I know what you tried to mean, but I guess it lacks a bit of usage and define the scope.

Consider:
 - if you do have "sudo" as part of os-core, then you:
   * remove it from the bundle and publish the update, *all* the devices will have sudo removed.
   * add it from the bundle and publish the update, *all* the devices will have sudo installed.
 - if you do have "sudo" as "sudo_bundle", then you do not automatically install or remove that in any device. You only make it available to be installed on each device, that have an option to do or not.

Then the purpose of the bundle is to allow a feature to be installed or removed *per device*. This can be exemplified as an application that one user may want and the other may not. Like a security system that may optionally install the upnp/dlna media provider to access the recorded files. Another example would be a drone, that can have a "camera" bundle installed that would be composed of the kernel module (v4l), media pipeline (gstreamer) and a server to stream it.